test/java/seedu/addressbook/commands/CountCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import java.util.Collections;
+import java.util.List;
+public class CountCommandTest {
+    private static final List<ReadOnlyPerson> EMPTY_PERSON_LIST = Collections.emptyList();
+    @Test
+
+    public void countCheckZero() throws Exception {
```

Comment:
Coding standard requires to include descriptive header comments for public methods